File: sdks/thunkmetrc-ts/wrapper/src/models/Result.ts

export interface Result {
    ExpirationDateTime?: string;
    LabFacilityLicenseNumber?: string;
    LabFacilityName?: string;
    LabTestDetailRevokedDate?: string;
    LabTestResultDocumentFileId?: number;
    LabTestResultId?: number;
    OverallPassed?: boolean;
    PackageId?: number;
    ProductCategoryName?: string;
    ProductName?: string;
    ResultReleaseDateTime?: string;
    ResultReleased?: boolean;
    RevokedDate?: string;
    SourcePackageLabel?: string;
    TestComment?: string;
    TestInformationalOnly?: boolean;
    TestPassed?: boolean;
    TestPerformedDate?: string;
    TestResultLevel?: number;
    TestTypeName?: string;
}

File: sdks/thunkmetrc-py/wrapper/src/thunkmetrc/wrapper/models/result.py
from typing import TypedDict, List, Optional, Any, Generic, TypeVar, TYPE_CHECKING
from typing_extensions import TypedDict

if TYPE_CHECKING:
    from . import *

class Result(TypedDict, total=False):
    ExpirationDateTime: str
    LabFacilityLicenseNumber: str
    LabFacilityName: str
    LabTestDetailRevokedDate: str
    LabTestResultDocumentFileId: int
    LabTestResultId: int
    OverallPassed: bool
    PackageId: int
    ProductCategoryName: str
    ProductName: str
    ResultReleaseDateTime: str
    ResultReleased: bool
    RevokedDate: str
    SourcePackageLabel: str
    TestComment: str
    TestInformationalOnly: bool
    TestPassed: bool
    TestPerformedDate: str
    TestResultLevel: float
    TestTypeName: str
